### PR TITLE
Bumping ursa to 0.9.4

### PR DESCRIPTION
**Pull request by mwain**
ursa 0.9.4 has fixes for node 6 deprecation warnings and efforts to fix the issues with installing it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ node_js:
   - 0.10
   - 0.12
   - 4.0
+  - 6.0
```
Comment (herom) [inline .travis.yml]: Please revert this change as it's not obvious why this would change with a version bump of a dependency -  feel free to open up a new PR in order to get tests for node.js 6.x :+1:

File: test/_socketStream.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+'use strict';
```
Comment (herom) [inline test/_socketStream.js]: The same as above, please revert this change as it's not obvious why this would be changed in the same commit as a dependency version bump.

**Comment by mwain** [comment]
@herom I have updated this PR to remove the other changes.

**Comment by herom** [comment]
Thanks a lot :+1:
